configs/nationwide_gh_config.yaml

```diff
@@ -24,6 +24,9 @@ graphhopper:
     - name: truck
       vehicle: truck
       weighting: fastest
+    - name: small_truck
```

Comment:
I realize it might not be super necessary to add the profile to this config (which is just used for generating the street export), but I like to have all vehicles we actually route with included for peace of mind (if we route on a link with any of our vehicles, we want it included in the export)